src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -13904,6 +13904,20 @@ namespace System.Runtime.InteropServices.Swift
         public unsafe SwiftSelf(void* value) { throw null; }
         public unsafe void* Value { get { throw null; } }
     }
+    [System.CLSCompliantAttribute(false)]
+    public readonly partial struct SwiftSelf<T> where T: unmanaged
+    {
+        private readonly int _dummyPrimitive;
+        public unsafe SwiftSelf(T value) { throw null; }
+        public unsafe T Value { get { throw null; } }
+    }
+    [System.CLSCompliantAttribute(false)]
+    public readonly partial struct SwiftIndirectResult
+    {
+        private readonly int _dummyPrimitive;
```

Comment:
@tannergooding or @jaredpar Is this needed for this type? I can't remember all the reasons for these dummy types, but this seems like a blittable type so I don't think this is ever needed, right?

Reply:
Yes, this is needed to let Roslyn know that it is unmanaged type with private fields.

If this field was delete, the following would compile successfully:

```
SwiftIndirectResult M()
{
    SwiftIndirectResult x;
    return x;
}
```

With this field present, this will fail to compile with ` error CS0165: Use of unassigned local variable 'x'` error.

Reply:
Indeed. Fields that are `private` on a `struct` can have significant impact on how the C# compiler treats them. 

https://blog.paranoidcoding.org/2016/02/15/are-private-members-api-surface.html